test/lit/wasm-split/multi-split-escape-names.wast

```diff
@@ -20,13 +22,13 @@
 
  ;; MOD1:      (type $2 (func (result i32)))
 
- ;; MOD1:      (import "" "table" (table $timport$0 1 funcref))
+ ;; MOD1:      (import "" "table" (table $timport$0 3 funcref))
 
- ;; MOD1:      (import "" "std::operator<<\\28std::__2::basic_ostream<char\\2c\\20std::__2::char_traits<char>>&\\2c\\20wasm::Module&\\29" (func $std::operator<<\28std::__2::basic_ostream<char\2c\20std::__2::char_traits<char>>&\2c\20wasm::Module&\29 (result f32)))
+ ;; MOD1:      (import "" "trampoline_std::operator<<\\28std::__2::basic_ostream<char\\2c\\20std::__2::char_traits<char>>&\\2c\\20wasm::Module&\\29" (func $trampoline_std::operator<<\28std::__2::basic_ostream<char\2c\20std::__2::char_traits<char>>&\2c\20wasm::Module&\29 (result f32)))
```

Comment:
Can we split the unrelated improvements like adding "trampoline" prefixes to names out into preliminary PRs to minimize test churn? It would also be nice if we can avoid changing the order of trampoline functions in the output (but I can imagine that might be more effort than it's worth).